package.json

```
@@ -39,9 +39,10 @@
   "scripts": {
     "bump": "beachball bump",
     "change": "beachball change",
-    "check": "beachball check",
+    "checkchange": "beachball check  --scope \"packages/*\" --changehint \"Run 'yarn change' to generate a change file\"",
```

I was playing around with this locally - I think what we want is below. It should check for the presence of change files in all packages but not the sites, which would be what we want IMO. If there are changes *only* to sites, we don't need change files. Any changes under packages AND those would require them.

```suggestion
    "checkchange": "beachball check --scope \"!sites/*\" --changehint \"Run 'yarn change' to generate a change file\"",
```

---

For some reason passing in packages/ explicitly doesn't work, so I the above is correct.